### PR TITLE
Release 4.1.1

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.1.1 (2020-09-25)
+
+- Fixed the avatar component to properly set accessibility labels and text for screen reader users. [#44](https://github.com/blackbaud/skyux-avatar/pull/44)
```
Comment (Blackbaud-AlexKingman) [inline CHANGELOG.md]: Maybe just say `screen readers` ? Not sure if we already have an established pattern for that...

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-release-4.1.1](https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-release-4.1.1)

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: I’m fine with that. Changed.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-release-4.1.1](https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-release-4.1.1)

**Review by Blackbaud-DeniseCisneros — APPROVED** [review]

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**